debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java

```diff
@@ -852,6 +854,11 @@ public Map<String, String> configs() {
                                 int maxRetries = getErrorsMaxRetries();
                                 LOGGER.info("Retriable exception thrown, connector will be restarted; errors.max.retries={}", maxRetries, e);
                                 if (maxRetries != 0) {
+                                    if (maxRetries < DEFAULT_ERROR_MAX_RETRIES) {
```

Comment:
maybe move outside the if you introduced:
if (maxRetries < DEFAULT_ERROR_MAX_RETRIES) {
...
}
else if (maxRetries != 0) {
...